src/formio/components/addressNL.ts

```diff
@@ -8,6 +8,9 @@ export interface AddressData {
   houseNumber: string;
   houseLetter: string;
   houseNumberAddition: string;
+  city: string;
+  streetName: string;
+  secretStreetCity: string;
```

Comment:
```suggestion
  city?: string;
  streetName?: string;
  secretStreetCity?: string;
```
these can (and will) be absent when `deriveAddress` is false, no?